docs/kb/rules/understand-severities.md

```diff
@@ -9,7 +9,7 @@ tags:
 
 ## Semgrep Code and Secrets
 
-Semgrep Code and Secrets rules have one of three severity levels: `ERROR` (High), `WARNING` (Medium), or `INFO` (Low). The severity indicates how critical the issues that a rule potentially detects are.
+Semgrep Code and Secrets rules have one of four severity levels: Critical, High, Medium, and Low. The severity indicates how critical the issues that a rule potentially detects are.
```

Comment:
(Sorry for getting to this so late!) - curious, do we not display `ERROR` and `WARNING` in the SARIF or JSON or API?

Reply:
Not the API, but in the JSON and SARIF outputs, yes. But it looks like those are kept as-is to be backward compatible. Do you think I should update and mention this?

Reply:
Maybe I should mention it in the JSON/SARIF doc instead. Let's ship this as is, it looks cleaner. Feel free to resolve the comment!